src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -629,18 +629,48 @@ void ov::npuw::CompiledModel::implement_properties() {
     // that can be used later to return requested properties by user.
     // It does it in 3 steps:
     //
-    // 1. Create mappings for all coped from HETERO plugin properties.
+    // 1. Create mappings for all coped from HETERO plugin properties +
+    //    OV public properties and hints as in ::intel_npu::CompiledModel.
     // 2. Fill `m_all_supported_props` with properties from 1st step.
     // 3. Create mappings for all NPUW-specific properties to getters of their
     //    values from config.
 
+#define GET_PUBLIC_PROP(property)                                                                     \
```

Comment:
With most of the properties above why do we need this for a specific subgraph? I mean can't we just get it for the initial full model?

Reply:
Thank you! The issue is in that that It is not compiled. I have to take properties from compiled model. This is why I use submodel for that!

Reply:
Sorry, I didn't get you